### PR TITLE
Add Contact UI( Controller, ContactHome, Contactreponseconfrm)

### PR DESCRIPTION
**Pull request by shamreenshaik**
Contact UI

### CODE REVIEW

File: CrisisApplication/Startup.cs [hunk 1]
```diff
@@ -42,7 +42,7 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env)
             app.UseStaticFiles();
             app.UseStatusCodePages();
             app.UseMvc(routes => {
-                routes.MapRoute(name: "default", template: "{controller=Account}/{action=SignIn}/{id?}");
+                routes.MapRoute(name: "default", template: "{controller=Contact}/{action=ContactHome}/{id?}");
```
Comment (shailscript) [inline CrisisApplication/Startup.cs]: Why do we need to make Contact UI the default route?

File: CrisisApplication/Views/Contact/Contactresponseconfrm.cshtml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+﻿<body class="jumbotron-fluid">
+<div class="jumbotron text-center">
+    <h1 class="display-3">Thank You!</h1>
+    <p class="lead"><strong>Your response has been recorded.</strong> </p>
+    <hr>
+    <p>
+        Having trouble? <p> Contact us info@crisimanager.ca</p>
```
Comment (shailscript) [inline CrisisApplication/Views/Contact/Contactresponseconfrm.cshtml]: I'd recommend you use a working website or email. 
Ex: `<a href="mailto:sshukl15@my.centennialcollege.ca">Crisis Manager App Email</a>`

**Review by shailscript — COMMENTED** [review]
Left some comments for you. Please address them.

File: CrisisApplication/Startup.cs (hunk 1 above)
Comment (shamreenshaik) [reply CrisisApplication/Startup.cs]: will change to signIn

File: CrisisApplication/Views/Contact/Contactresponseconfrm.cshtml (hunk 2 above)
Comment (shamreenshaik) [reply CrisisApplication/Views/Contact/Contactresponseconfrm.cshtml]: okay

File: CrisisApplication/Views/Contact/Contactresponseconfrm.cshtml [hunk 3]
```diff
@@ -0,0 +1,14 @@
+﻿<body class="jumbotron-fluid">
+<div class="jumbotron text-center">
+    <h1 class="display-3">Thank You!</h1>
+    <p class="lead"><strong>Your response has been recorded.</strong> </p>
+    <hr>
+    <p>
+        Having trouble? <p> Contact us at ssadiq1@my.centennialcollege.ca</p>
```
Comment (shailscript) [inline CrisisApplication/Views/Contact/Contactresponseconfrm.cshtml]:
```suggestion
        Having trouble?<p> Contact us <a href="mailto:ssadiq1@my.centennialcollege.ca">here</a></p>
```

**Review by shailscript — APPROVED** [review]
Make sure we commit this suggestion before merging in.
Otherwise, looks good! 👍

**Pull request merged**